projects/packages/changelogger/composer.json

```
@@ -15,7 +15,7 @@
 		"symfony/process": "^3.4 || ^4.4 || ^5.2 || ^6.0 || ^7.0"
 	},
 	"require-dev": {
-		"yoast/phpunit-polyfills": "^1.1.1",
+		"yoast/phpunit-polyfills": "^1.1.1 || ^2.0 || ^3.0",
```

We should be fine with just
```suggestion
		"yoast/phpunit-polyfills": "^3.0",
```
The trickier part is making sure everything actually works with PHPUnit 11.